tests/cephfs/cephfs_utils.py

```diff
@@ -29,15 +29,55 @@ def __init__(self, ceph_cluster):
         self.rc_list = []
         self.mons = ceph_cluster.get_ceph_objects("mon")
         self.mgrs = ceph_cluster.get_ceph_objects("mgr")
+        self.__install_python3()
+
+    def __install_python3(self):
+        """
+        Installation of python3 and pip3
+        """
+        clients = self.ceph_cluster.get_ceph_objects("client")
+
+        for client in clients:
+            # Install python3
+            client.exec_command(
+                sudo=True,
+                cmd="yum install -y python3",
+                long_running=True,
+                check_ec=False,
+            )
+
+            # Install pip3
+            client.exec_command(
+                sudo=True,
+                cmd="yum install -y python3-pip",
+                long_running=True,
+                check_ec=False,
+            )
```

Comment:
can you try to combine thes two. lie `packages  = ['python3', 'pip3']` and iterate over.

Reply:
Hi Rakesh,

I have created a list and iterating through it